super_user/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Feel free to delete these extra files created by Django scaffolding if you're not using them.